scripts/coins-logon-widget.js

```diff
@@ -1,115 +1,79 @@
 'use strict';
-var EventEmitter = require('wolfy87-eventemitter');
-var assign = require('lodash/object/assign');
-var cloneDeep = require('lodash/lang/cloneDeep');
+
 var auth = require('./lib/auth');
-var Form = require('./lib/form');
-var FormGroup = require('./lib/form-group');
+var bind = require('lodash/function/bind');
+var createElement = require('virtual-dom/create-element');
+var diff = require('virtual-dom/diff');
+var EventEmitter = require('wolfy87-eventemitter');
+var form = require('./components/form');
+var merge = require('lodash/object/merge');
+var messages = require('./lib/messages');
+var patch = require('virtual-dom/patch');
+var uniqueId = require('lodash/utility/uniqueId');
 var utils = require('./lib/utils');
-require('./lib/polyfills.js');
```

Comment:
Using lodash’s `bind` to minimize external deps. Although…eh, it doesn’t really matter. We can drop this when PhantomJS 2.0.0 ships to NPM.